database/insertData/laos/02_template_orgRegistration.js

```diff
@@ -0,0 +1,1003 @@
+                      isRequired: false
+                      parameters: {
+                        label: "Fax number"
+                        description: "(if applicable)"
+                      }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^[0-9()\\\\-\+]*$"
+                        ]
+                      }
+                      validationMessage: "Not a valid fax number"
+                    }
+                    {
+                      code: "email"
+                      index: 90
+                      title: "Email"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: {
+                        label: "Email"
+                        default: {
+                          operator: "objectProperties"
+                          children: ["currentUser.email"]
+                        }
+                      }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          {
+                            value: "^[A-Za-z0-9.]+@[A-Za-z0-9]+\\\\.[A-Za-z0-9.]+$"
+                          }
+                        ]
+                      }
+                      validationMessage: "Not a valid email address"
+                    }
+                    {
+                      code: "PBOwner"
+                      index: 100
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                    }
+                    {
+                      code: "ownerIntro"
+                      index: 105
+                      title: "Owner Intro"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      helpText: "This page requires information about the **owner** of the company, which will probably be the person filling in this application form"
+                      parameters: { title: "### Company Owner", style: "none" }
+                    }
+                    {
+                      code: "ownerName"
+                      index: 110
+                      title: "Company Owner"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: {
+                        label: "Full name of company owner"
+                        description: "TO-DO: Pre-populate"
```

Comment:
I think should either remove this TODO - so this is just a regular text field for now - or enter the current user First name here and all next fields need to be captured from "User registration" which will need an update to also have Phone and fax

Reply:
I'm working on this now.